python-threatexchange/threatexchange/signal_type/tests/test_hash_from_x.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+import os
+
+from threatexchange.content_type import meta
+from threatexchange.signal_type.signal_base import FileHasher, StrHasher
+from pathlib import Path
+
+
+class TestHashFrom(unittest.TestCase):
```

Comment:
Nit, ignorable: SignalTypeHashTest or TestHash or something similar, TestHashFrom (and having the reader put together that this is a prefix of methods on SignalType might be tough.